src/devtools/client/webconsole/reducers/messages.ts

```diff
@@ -285,6 +282,20 @@ const messagesSlice = createSlice({
         state as MessageState
       );
     },
+    clearMessages(state) {
+      removeMessagesFromState(state as MessageState, state.messages.ids as string[]);
+      state.overflow = false;
+      state.messagesLoaded = false;
+    },
```

Comment:
This looks unnecessarily slow, since it's doing iteration and individual deletion:
https://github.com/RecordReplay/devtools/blob/66136c37002ab85c152b477e41252351f012d959/src/devtools/client/webconsole/reducers/messages.ts#L443-L459

I think we can just clear and recreate both the id and entities object directly.